maint/code-cleanup.sh

```diff
@@ -10,10 +10,56 @@ indent_code()
 {
     file=$1
 
-    $indent --k-and-r-style --line-length100 --else-endif-column1 --start-left-side-of-comments \
-	--break-after-boolean-operator --dont-cuddle-else --dont-format-comments \
-	--comment-indentation1 --indent-level4 --no-tabs \
-	${file}
+    $indent \
+        \ # Expansion of Kernighan & Ritchie style
```

Comment:
So my `bash` doesn't like these inline style comments and won't run the script. Stackoverflow recommends using backticks...